lib/server/push.api.js

```diff
@@ -71,15 +80,13 @@ Push.Configure = function (options) {
     console.log('Push: APN configured');
   }
 
-  const apn = Npm.require('apn');
-
   const useProductionCert = process.env.NODE_ENV === 'production';
 
   if (!useProductionCert) {
     console.log('This is being configured in development mode');
   }
 
-  const vets = GlobalVets.find({$or: [{disabled: {$exists: false}}, {disabled: false}]}).fetch();
+  const vets = GlobalVets.find({$or: [{disabled: {$exists: false}}, {disabled: false}], useFirebaseForMessaging: true}).fetch();
```

Comment:
@mjovkovic Do we need `if (daysRemaining > 1)` check? As we are not using certificates anymore. Right?

Reply:
That's true!